### ISSUE TITLE
download links

### ISSUE THREAD

**Issue by ivylinden** [PKU-IMRE/VERI-Wild#11]
Hi,  I have read your CVPR paper and I want to test my model on  this dataset. But I can't find the available download links?

**Comment by ngunauj** [comment]
+1

**Comment by CHWangXing** [comment]
+1

**Comment by daiguangzhao** [comment]
hi, cloud you put your dataset up to your GitHub

**Comment by PKU-IMRE** [comment]
The download link has been provided in github.

**Comment by yanbai1993** [comment]
Download link for VERI-Wild:
Baidu cloud disk: Link: https://pan.baidu.com/s/1ib_gj5IKI2hSv1RbGsQmFwPassword: fbm1
Google driven: https://drive.google.com/drive/folders/1q9kClC0gy0bz2i06sHlKUh2Pqk6RxKan?usp=sharing



Download link for VERI-Wild 2.0:

VERI-Wild 2.0 has a challenging and large test set containing about 400K vehicle images that do not have any camera overlap with the training set. VERI-Wild 2.0 shares the same training set with VERI-Wild.
Baidu cloud disk: Link: https://pan.baidu.com/s/186nLRk-7X2RssK8cTDCQPw Password: qz32
Google driven: https://drive.google.com/drive/folders/1rfuIs1xSCa7CN6IKvMOzjQfbqjY-7E0h?usp=sharing
